### PR TITLE
Tag licences to industries and locations

### PR DESCRIPTION
**Pull request by leenagupte**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

Trello: https://trello.com/c/DY5YCALU
Related to: https://github.com/alphagov/licence-finder/pull/1275

# What's changed?
Updates the Licence Transaction import script to also tag licences their industry and location facets.
The licences are tagged to the level sectors that they were previously tagged to in Licence Finder.

# Why?

Facets in specialist finders do not support nested facets, therefore only one level of sectors was required, and level two was selected by the user researchers and content designers for the first round of testing.

The tagging is being done programmatically as there are approximately 420 imported licences that need tagging.

The licences are being tagged to the same industries (sectors) that were tagged to in Licence Finder so that it's easier to test that the licences have been tagged correctly.

Any changes to industry facets (i.e. names / values) will occur outside of this change.

# Results from testing
Successful run of import script in [staging](https://deploy.blue.staging.govuk.digital/job/run-rake-task/218499/console).

Screenshot 2023-02-02 at 13 54 15

### CODE REVIEW

File: lib/importers/licence_transaction/facet_tagger.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require "uri"
+require "net/http"
+
+module Importers
+  module LicenceTransaction
+    class FacetTagger
```
Comment (1pretz1) [inline lib/importers/licence_transaction/facet_tagger.rb:6]: Nice job separating this out from the main importer class

File: lib/importers/licence_transaction/facet_tagger.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+require "uri"
```
Comment (1pretz1) [inline lib/importers/licence_transaction/facet_tagger.rb:1]: This commit could be squashed into the previous one but not a blocker 👍

File: lib/importers/licence_transaction/facet_tagger.rb [hunk 3]
```diff
@@ -28,8 +28,10 @@ def licence_finder_api_data
       end
 
       def licence_finder_api_response
-        response = Net::HTTP.get_response(licence_finder_api_url).body
-        JSON.parse(response)
+        Rails.cache.fetch(licence_finder_api_url, expires: 1.hour) do
```
Comment (1pretz1) [inline lib/importers/licence_transaction/facet_tagger.rb:31]: Awesome!

**Review by 1pretz1 — APPROVED** [review]
Nice work - looks good to me 💯

File: lib/importers/licence_transaction/facet_tagger.rb (hunk 2 above)
Comment (leenagupte) [reply lib/importers/licence_transaction/facet_tagger.rb:1]: This class doesn't exist in the previous commit ☺️

**Pull request merged**